pkg/rpcserver/rpcserver.go

```diff
@@ -51,6 +51,9 @@ import (
 
 func init() {
 	grpc.EnableTracing = false
+	for rpc, paths := range ttnpb.AllowedFieldMaskPathsForRPC {
+		validator.RegisterAllowedFieldMaskPaths(rpc, paths...)
```

Comment:
Shouldn't this happen in `cmd/ttn-lw-stack` instead?
Feels weird to hardcode allowed paths from a package with such a generic name.

Reply:
This is the one point where we already work with the both the `validator` package and `ttnpb`. We could move it in the future if we think that would be better.